cmd/internal/codegen/parse/crd.go

```diff
@@ -410,9 +444,38 @@ func getValidation(comment string, props *v1beta1.JSONSchemaProps) {
 		}
 		props.MultipleOf = &f
 	case "Enum":
-		enums := strings.Split(parts[1], ",")
-		for i := range enums {
-			props.Enum = append(props.Enum, v1beta1.JSON{[]byte(enums[i])})
+		if props.Type != "array" {
+			value := strings.Split(parts[1], ",")
```

Comment:
I'm wondering if we can separate this code block of parsing Enum to a different function.  Then the function `getValidation` will keep short and we can also test `Enum` alone.

Reply:
Agree. I wanted to do ti as well. I have encapsulated it into util functions.